pkg/metrics/metrics.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/gorilla/mux"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Un-needed alias (also, perhaps better to alias to `corev1` to match other parts of codebase?)